tests/utils.go

```diff
@@ -1934,17 +1951,27 @@ func NewRandomFedoraVMIWithDmidecode() *v1.VirtualMachineInstance {
 	return vmi
 }
 
-func GetGuestAgentUserData() string {
+func GetGuestAgentUserData(ipv6Config *IPv6Config) string {
```

Comment:
Create a function to compose ipv6 net configuration commands.

```golang
ComposeIpv6NetCommands(IPv6Config) string
```

Then you have
```golang
GetGuestAgentUserData()
GetGuestAgnetUserDataWithExtraComamnds(string)
```
Then you will use it like

```golang
 GetGuestAgentUserDataWithExtraComands(Composeipv6NetCommands(ipv6Config))
``` 

This way you don't need the if statement.